api/Worker.json

```diff
@@ -163,6 +163,43 @@
             }
           }
         },
+        "enforces_trusted_types": {
+          "__compat": {
+            "description": "`url` parameter requires `TrustedScriptURL` instance when trusted types are enforced",
```

Comment:
Tested with https://wpt.live/trusted-types/DedicatedWorker-constructor-from-DedicatedWorker.html

Note, I didn't nest this as a property of an URL sub feature. WOuld you prefer that? Ditto for the SharedWorker

Reply:
```suggestion
            "description": "Requires `url` parameter to be `TrustedScriptURL` instance when trusted types are enforced",
```